assembl/fabfile.py

```diff
@@ -794,6 +801,155 @@ def update_pip_requirements(force_reinstall=False):
         run("yes w | %s" % cmd)
 
 
+def is_db_updated():
+    """
+    Return if the database is update or not
+    """
+    history = venvcmd('alembic -c %s history' % (env.ini_file))
+    current = venvcmd('alembic -c %s heads' % (env.ini_file))
+    return current in history
+
+
+@task
+def reset_db():
+    """
+    Restore and update the latest database
+    """
+    # Only for the staging server (for tests)
+    if env.wsginame == 'staging.wsgi':
+        exists = False
+        # Test if the dump exists on the amazon s3 object storage
+        with shell_env(
+            AWS_ACCESS_KEY_ID=env.aws_access_key_id,
+            AWS_SECRET_ACCESS_KEY=env.aws_secret_access_key
+        ):
+            exists = venvcmd('aws s3 ls s3://%s/%s | wc -l' % (env.aws_bucket_name, get_db_dump_name()))
```

Comment:
I tend to prefer .format() more readable, do we have a guideline for that?

Reply:
I agree with you. the fab file uses this syntax ... Done.